chat-plugins/datasearch.js

```diff
@@ -122,6 +122,7 @@ exports.commands = {
 		`Search categories are: type, tier, color, moves, ability, gen, resists, recovery, priority, stat, weight, height, egg group.`,
 		`Valid colors are: green, red, blue, white, brown, yellow, purple, pink, gray and black.`,
 		`Valid tiers are: Uber/OU/BL/UU/BL2/RU/BL3/NU/BL4/PU/NFE/LC/CAP/CAP NFE/CAP LC.`,
+		`Valid doubles tiers are: DUber/DOU/DBL/DUU/DNFE.`,
```

Comment:
DNFE is not in fact a valid doubles tier

Reply:
I know, but thats how you search for mons that appear as nfe in doubles

Reply:
What does "appear as NFE in doubles" mean? All Pokémon that are NFE in singles are NFE in doubles...?

Reply:
The description is technically "NFEs not in a higher tier", and the NFEs in higher tiers are different in singles than in doubles. For example, Porygon2 is DUU in Doubles and RU in Singles, making it an NFE in a higher tier for both formats. However, Golbat is NU in Singles but NFE in Doubles, meaning that it falls under "NFEs not in a higher tier" for Doubles but not Singles.

Reply:
I don't like this definition of "NFE" and I don't consider it useful, and it also contracts what "NFE" actually stands for.

"NFE" as a pokemon tag should include all NFEs, whether or not their `tier === 'NFE'`. Just like "LC" as a pokemon tag should include all pokemon legal in LC, whether or not their `tier === 'LC'`.

Reply:
removed